src/boot/efi/proto/cc-measurement.h

```diff
@@ -0,0 +1,60 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+#pragma once
+
+#include "efi.h"
+
```

Comment:
Please add some references here as a comment, as it's not the most well known interface

Reply:
added a short description.